decidim/decidim
# Display only once resource type in search

#### :tophat: What? Why?
This PR ensures that the search page is displaying only once the resources that are being filterable in the search page. 

It will convert : 
![image](https://github.com/decidim/decidim/assets/105683/44b2e66b-460c-4c03-b5bf-ec1177f970fe)

To something smaller like: 
![image](https://github.com/decidim/decidim/assets/105683/4f2e8086-d3d3-49c5-b51d-ceb2e521d2b5)

#### Testing
Make sure the pipeline is green on the relevant actions. 
1. Visit the frontend application 
2. Search for an empty string : http://localhost:3000/search?term=
3. See the list 
4. Apply patch 
5. See the new list ( you may need to restart the application )

### :camera: Screenshots
*Please add screenshots of the changes you are proposing*
try.decidim.org 
![image](https://github.com/decidim/decidim/assets/105683/fef52adb-3f09-4bda-9800-7a579ed57e02)

nightly.decidim.org
![image](https://github.com/decidim/decidim/assets/105683/44b2e66b-460c-4c03-b5bf-ec1177f970fe)

Patched version; 
![image](https://github.com/decidim/decidim/assets/105683/4f2e8086-d3d3-49c5-b51d-ceb2e521d2b5)

:hearts: Thank you!
